### PR TITLE
Vinny/bbo bug fix

### PR DESCRIPTION
**Pull request by vinayakSharm**
bbo bug fix

### CODE REVIEW

File: fonpr/agent_bbo.py [hunk 1]
```diff
@@ -139,17 +143,23 @@ def update_yml(
     # github yml file url that controls app to be modified
     gh_url = "https://github.com/DISHDevEx/napp/blob/aakash/hpa-nodegroups/napp/open5gs_values/5gSA_no_ues_values_with_nodegroups.yaml"
 
-    wait_time = 3600
+    wait_time = 5
```
Comment (PaperArmada) [inline fonpr/agent_bbo.py:146]: Why the switch to 5 seconds?

Comment (vinayakSharm) [reply fonpr/agent_bbo.py:146]: for testing purposes!! let me revert back to 3600 and push

Comment (vinayakSharm) [reply fonpr/agent_bbo.py]: changed back to 3600!

**Review by PaperArmada — APPROVED** [review]
Docker build successful; lgtm

**Pull request merged**